File: homemarket-domain/src/main/java/com/plateer/homemarket/entity/Order.java
package com.plateer.homemarket.entity;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class Order {
	
	private int orderId;
	private int productId;
	private String productName;
	private int count;
	private String state;
	private String address;
	private String memberId;
	private String orderDate;
}

File: vue/src/model/Order.js
class Order{
    constructor(orderId, productId, productName, count, state, address, memberId, orderDate){
        this.orderId = orderId,
        this.productId = productId,
        this.productName = productName,
        this.count = count,
        this.state = state,
        this.address = address,
        this.memberId = memberId,
        this.orderDate = orderDate
    }
}

export default Order;